LICENSE

```diff
@@ -1,7 +1,8 @@
 Copyright (c) 2012 Vladimir Keleshev <vladimir@keleshev.com>
-                   Blake Williams <code@shabbyrobe.org>
-                   Alex Speller <alex@alexspeller.com>
-                   Nima Johari
+                   Eyal Shalev <eyalsh@gmail.com>
+                   Blake Williams <code@shabbyrobe.org>	
+                   Alex Speller <alex@alexspeller.com>	
+                   Nima Johari	
```

Comment:
Looking at [docopt.go/LICENSE](https://github.com/docopt/docopt.go/blob/master/LICENSE) It seems you were correct
```suggestion
Copyright (c) 2020 Eyal Shalev <eyalsh@gmail.com>
```